<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profilePage</class>
 <widget class="QDialog" name="profilePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbUserId">
     <property name="text">
      <string>Идентификатор:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbUserLogin">
     <property name="text">
      <string>Логин:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbUserFullName">
     <property name="text">
      <string>Полное имя:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbUserRole">
     <property name="text">
      <string>Роль:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbUserClassId">
     <property name="text">
      <string>Класс:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
